cpp/src/slice2cs/Gen.cpp

```diff
@@ -528,12 +528,13 @@ Slice::CsVisitor::writeValue(const TypePtr& type, const string& ns)
 }
 
 void
-Slice::CsVisitor::writeConstantValue(const TypePtr& type, const SyntaxTreeBasePtr& valueType, const string& value)
+Slice::CsVisitor::writeConstantValue(const TypePtr& type, const SyntaxTreeBasePtr& valueType, const string& value,
+    const string& ns)
 {
     ConstPtr constant = ConstPtr::dynamicCast(valueType);
-    if(constant)
+    if (constant)
```

Comment:
Probably shouldn't be a space here, even if it is generating C# code!

Reply:
the space is correct.